packages/webapp-manager/webapp-manager.pacscript

```diff
@@ -1,11 +1,15 @@
 name="webapp-manager"
-version="1.2.8"
-url="https://github.com/linuxmint/webapp-manager/archive/refs/tags/${version}.tar.gz"
+version="1.3.0"
```

Comment:
Don't have 2 versions if you can.

Reply:
See my comment 

> Version 1.3.0 gives this error in jammy:
> 
> ```
>   File "/usr/lib/webapp-manager/webapp-manager.py", line 24, in <module>
>     from common import _async, idle, WebAppManager, download_favicon, ICONS_DIR, BROWSER_TYPE_FIREFOX
>   File "/usr/lib/webapp-manager/common.py", line 397, in <module>
>     def download_image(root_url: str, link: str) -> Optional[Image]:
>   File "/usr/lib/python3.10/typing.py", line 312, in inner
>     return func(*args, **kwds)
>   File "/usr/lib/python3.10/typing.py", line 403, in __getitem__
>     return self._getitem(self, parameters)
>   File "/usr/lib/python3.10/typing.py", line 529, in Optional
>     arg = _type_check(parameters, f"{self} requires a single type.")
>   File "/usr/lib/python3.10/typing.py", line 176, in _type_check
>     raise TypeError(f"{msg} Got {arg!r:.100}.")
> TypeError: typing.Optional requires a single type. Got <module 'PIL.Image' from '/usr/lib/python3/dist-packages/PIL/Image.py'>.
> ```

Reply:
So then 1.3.0 doesn't work on Jammy, so it's incompatible.

Reply:
What if I am on jammy and I want to install webapp-manager with pacstall?

Reply:
I don't like the idea of more packages starting to use different versions in the same script.

Reply:
Can we create scripts like webapp-manager-legacy, which would keep an older, more compatible version?

Reply:
Sure!